app/models/config.rb

```diff
@@ -0,0 +1,7 @@
+class Config < ApplicationRecord
```

Comment:
@jacobherrington would it make sense to move these in an `models/internal` namespace/folder?